packages/ui-kit/src/components/Logger/Logger.stories.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { storybookCodeTemplate } from '../../helpers'
+import { LogProvider } from './LogProvider'
+import { useLog } from './useLog'
+import { LogLevel } from './Log.types'
+
+const ChildComponent: React.FC<unknown> = () => {
+  const log = useLog()
+
+  const level = log.level.toUpperCase()
```

Comment:
Can we change this to
```suggestion
  const level = JSON.stringify(log.level)
```

It's better if this prints as `"info"`, `"warn"`, etc., so that the messages we see are

> This is an error-level message with the log level set to "info".
> This is a warn-level message with the log level set to "info".
> This is an info-level message with the log level set to "info".